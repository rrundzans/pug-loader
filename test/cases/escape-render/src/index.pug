h1 Hello world!

p "double quotes"
p 'single quotes'
p `backtick`

p.
  Newline string,
  next newline

p Add strings:
  | string1 with |
  | string2 with |