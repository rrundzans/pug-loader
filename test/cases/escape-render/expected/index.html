<h1>Hello world!</h1><p>"double quotes"</p><p>'single quotes'</p><p>`backtick`</p><p>Newline string, next newline</p><p>Add strings:string1 with | string2 with |</p>